raw_data_processor/src/main/scala/scalalab3/lyricsengine/reader/Reader.scala

```diff
@@ -0,0 +1,49 @@
+package scalalab3.lyricsengine.reader
+
+import com.typesafe.config.ConfigFactory
+
+import scala.io.Source
+
+object Reader {
+  val conf = ConfigFactory.load();
+
+  def main(args: Array[String]) {
+    print(extract())
+  }
+
+  def extract(): MSD = {
+    val path = conf.getString("path_to_msd")
+
+    val pair = read(path)
+
+    val words = extractWords(pair._1)
+    val songs = extractSongs(pair._2)
+
+    new MSD(words, songs)
+  }
+
+  private def read(path: String) = {
+    val lines = Source.fromFile(path).getLines().toSeq
+    val words = lines(17)
+    val songs = lines.slice(18, lines.size)
```

Comment:
Of course i can find data by pattern but, current approach have nice speed and little bit of code. Less code - more understanding.)
I see that our data have 'hand-made' format. I think, in this case, it's a good idea don't write complex loader.